.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "3rd-party/uniffi-rs"]
        path = 3rd-party/uniffi-rs
-       url = https://github.com/NordSecurity/uniffi-rs.git
```

Comment:
This is wrong if you are using NordSecurity fork

Reply:
fixed